### PR TITLE
Create light-control.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Move light control example to tutorials section of documentation..

### CODE REVIEW

**Comment by AnotherButler** [comment]
This is failing Travis. Could an engineer please look into this?

File: docs/tutorials/using_apis/connectivity/light-control.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+
+### Changing the radio driver
+
+To run a 6LoWPAN-ND network, you need a working RF driver for Nanostack. This example uses the Atmel AT86RF233 by default.
+
+To change the RF driver, modify the `mbed_app.json` file by setting preferred RF driver `provide_default` value to true, For example, to use MCR20a RF driver: 
+
+```json
+"atmel-rf.provide-default": false,
+"mcr20a.provide-default": true,
```
Comment (cmonr) [inline docs/tutorials/using_apis/connectivity/light-control.md]: I _think_ the Travis check is complaining that something is wrong with this JSON.

If I had to guess, the trailing comma on this line implies another key/value pair, which doesn't appear.

@geky Can you confirm?

**Comment by AnotherButler** [comment]
ping @geky

File: docs/tutorials/using_apis/connectivity/light-control.md (hunk 1 above)
Comment (geky) [reply docs/tutorials/using_apis/connectivity/light-control.md]: Yep, json is pedantic as usual.

**Comment by AnotherButler** [comment]
Muchas gracias, @geky 

@SeppoTakalo  @artokin Does this look OK to y'all?

**Review by SeppoTakalo — APPROVED** [review]

**Pull request merged**